main.yml

```diff
@@ -400,6 +400,44 @@ parameters:
           # Pull in the APPUiO managed info labels
           * on(cluster_id) group_left(sales_order) appuio_managed_info{
               cloud_provider=~"%(cloud_provider)s",
+              distribution=~"openshift4",
+              vshn_service_level="%(vshn_service_level)s",
+          }
+
+      appuio_managed_cores:
+        enabled: true
+        prometheus_org_id: appuio-managed-openshift-billing
+        products:
+          - product_id: 'openshiftoke-baremetal-core-besteffort'
+            params:
+              cloud_provider: baremetal
+              vshn_service_level: best_effort
```

Comment:
This needs to take into account `distribution` (which we don't have yet) so we don't incorrectly bill OCP4 on baremetal as OKE on baremetal.

Reply:
Distribution is currently hardcoded in the query, since it's constant anyway. Similarly I added "distribution=openshift4" in the query for vcpus.

If in the future we have more distributions that use the same query, I figured we can add a parameter then.

Reply:
nvm you raise a good argument for doing it now in your later comment (: